### PR TITLE
Blog app - controllers

### PR DESCRIPTION
**Pull request by ksigei**
In this milestone, I was able to:

- Create the following for each URL:

- [ ] -  A route
- [ ] - An action in the correct controller.
- [ ] - A view file.
- Add view files for each of the view

### CODE REVIEW

**Review by DammyShittu — COMMENTED** [review]
Hi @ksigei,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### Your linters have failed

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by ksigei** [comment]
@DammyShittu I am sorry, My mind is still on ruby where you just run rubocop

**Comment by ksigei** [comment]
@DammyShittu here is another PR 
https://github.com/ksigei/rails_blog/pull/6

File: config/routes.rb [hunk 1]
```diff
@@ -12,5 +12,12 @@
     end
   end
   
+  get '/posts', to: 'posts#index'
+  get '/posts/:id', to: 'posts#show'
+  get '/users', to: 'users#index'
+  get '/users/:id', to: 'users#show'
   get '/about', to: 'about#about'
+
```
Comment (arslanbisharat) [inline config/routes.rb:20]: [Optional] Typing the command `rails routes | grep user` should yield URLs that correspond to these wireframes
It is much better to write it this way: 
 ```
resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end

```

**Review by arslanbisharat — APPROVED** [review]
Hi @ksigei,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (ksigei) [reply config/routes.rb:20]: okay

**Pull request merged**